### ISSUE TITLE
pear.avalanche123.com is not available to serve the channel

### ISSUE THREAD

**Issue by benlk** [avalanche123/doxphp#14]
From the install instructions in README.md:

> pear channel-discover pear.avalanche123.com
> pear install avalanche123/doxphp-beta

Once upon a time, I set up this repo using `pear`. When I run `pear-channel-info pear.avalanche123.com`, I get the following response:

```
$ pear channel-info pear.avalanche123.com
Channel pear.avalanche123.com Information:
==========================================
Name and Server         pear.avalanche123.com
Alias                   avalanche123
Summary                 avalanche123's PEAR channel
Validation Package Name PEAR_Validate
Validation Package      default
Version
Server Capabilities
===================
Type Version/REST type Function Name/REST base
rest REST1.0           http://pear.avalanche123.com/rest/
rest REST1.1           http://pear.avalanche123.com/rest/
rest REST1.2           http://pear.avalanche123.com/rest/
rest REST1.3           http://pear.avalanche123.com/rest/
```

But none of those URLs exist; they all show a GitHub 404 page.

There does not appear to be an available alternate channel for this package, save for cloning directly from GitHub into `$PATH`.

**Comment by slertt** [comment]
Did u managed to solve it?

**Comment by benlk** [comment]
Yeah. We switched to installing doxphp via git, as documented in https://github.com/INN/largo/commit/16545dca23f583afaad08aa730e790f911a4735d#diff-6988d98ae8efd5611cbce8b016f16940 for https://github.com/INN/largo/issues/1456: 

Installation process with git:

1. `git clone https://github.com/avalanche123/doxphp.git` to someplace in your filesystem
2. add `doxphp/bin` in the cloned git repo to your `$PATH` by adding `export PATH=$PATH:/path/to/doxphp/bin` to your `.bashrc` or similar shell configuration file.
